src/nanoemoji/svg.py

```diff
@@ -149,7 +165,12 @@ def _glyph_groups(
 
                 nth_paint_glyph += 1
 
-    return reuse_groups.sorted()
+    sorted_groups = reuse_groups.sorted()
+
+    if has_color_notdef:
+        return ((".notdef",),) + sorted_groups
+    else:
+        return sorted_groups
```

Comment:
never liked this sort of branched return, wdyt about dropping the bool, storing a default-empty tuple of initial_glyphs, setting initial_glyphs to `(".notdef",)` above if there is one, and then here returning `return  (initial_glyphs,) + reuse_groups.sorted()`

Reply:
OK I can change that. But why don't you like them?